### DISCUSSION TITLE
unable to see the next slide using centeredSlides 'auto' in an infinite loop

### DISCUSSION THREAD

**Discussion by paolosax** [nolimits4web/swiper#7225] [Q&A]
Hello - after v.11 update I'm not able no more to get an infinite loop carousel with a centered slide and the previous/next slide always visible. What I'd like to get is to have the active sclide in the middle and half of the prev/next slides visible respectively on left & right of the active one. 
Now the carousel start correctly: 
- "half-left|center|half-right" 
but after the first animation, the next slide is not visible:
- "half-left|center|HALF-EMPTY".

This is the array of settings :

$swiper_settings = array(
            'slidesPerView' => 1,
            'centeredSlides' => true,
            'loop' => true,
            'freeMode' => true,
            'speed' => 1000,
            'autoplay' => array(
                'delay' => 2000,
                'disableOnInteraction' => false
            ),
            // 'normalizeSlideIndex' => false,
            'keyboard' => array(
                'enabled' => true,
                'onlyInViewport' => true
            ),
            'navigation' => array(
                'nextEl' => '.nav-btn--next',
                'prevEl' => '.nav-btn--prev'
            ),
            'pagination' => array(
                'el' => '.carousel-pagination',
                'clickable' => true
            ),
            'breakpoints' => array(
                '992' => array(
                    'slidesPerView' => 'auto',
                ),
            ),
            'effect' => 'slide',
        );
        
...this is a php array that will be json_encode(d)

Any suggestion are more than welcome...

**Comment by Reg93** [comment]
> Hello - after v.11 update I'm not able no more to get an infinite loop carousel with a centered slide and the previous/next slide always visible. What I'd like to get is to have the active sclide in the middle and half of the prev/next slides visible respectively on left & right of the active one. Now the carousel start correctly:
> 
> * "half-left|center|half-right"
>   but after the first animation, the next slide is not visible:
> * "half-left|center|HALF-EMPTY".
> 
> This is the array of settings :
> 
> $swiper_settings = array( 'slidesPerView' => 1, 'centeredSlides' => true, 'loop' => true, 'freeMode' => true, 'speed' => 1000, 'autoplay' => array( 'delay' => 2000, 'disableOnInteraction' => false ), // 'normalizeSlideIndex' => false, 'keyboard' => array( 'enabled' => true, 'onlyInViewport' => true ), 'navigation' => array( 'nextEl' => '.nav-btn--next', 'prevEl' => '.nav-btn--prev' ), 'pagination' => array( 'el' => '.carousel-pagination', 'clickable' => true ), 'breakpoints' => array( '992' => array( 'slidesPerView' => 'auto', ), ), 'effect' => 'slide', );
> 
> ...this is a php array that will be json_encode(d)
> 
> Any suggestion are more than welcome...

Hello. Did you fix issue?
